### PR TITLE
saints3.json

### PR DESCRIPTION
**Pull request by jatinparmar96**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

What changes does this PR introduce? 
Shows saints row 3 cheats
###### Related Issues or Discussions 

What issues or discussions (if any) does this PR address? Please provide the issue # or link.

###### People to notify

@mention any relevant people/organizations

---

Instant Answer Page: https://duck.co/ia/view/saints_row_3_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @mention

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by gautamkrishnar** [comment]
You have some conflict in your PR. Please fix it. If You are not sure about how to fix it, just open a new PR.  You must follow the guidelines mentioned here: https://docs.duckduckhack.com/submitting/pull-request.html

File: share/goodie/cheat_sheets/json/saint.json [hunk 1]
```diff
@@ -1,140 +1,141 @@
...
-            },
-             {
-                "val":"Vehicle Smash (smash vehicles easily)",
-                "key":"isquishyou"
-            }
-    ]
-    }
+{  
+   "id":"saints_row_cheat_sheet",
+   "name":"Saints Row IV Cheats",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/saint.json]: @jatinparmar96 If you are making this changes to add cheat sheets of two separate versions of the game, then you need to open separate PR for each version.

Comment (jatinparmar96) [inline share/goodie/cheat_sheets/json/saint.json]: Ok so should i close this request then?

Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/saint.json]: yes sure

**Pull request closed without merging**